sipa/model/sample/user.py

```diff
@@ -47,6 +48,7 @@ def init_app(app):
             'mail_confirmed': True,
             'mac': 'aa:bb:cc:dd:ee:ff',
             'ip': '141.30.228.39',
+            'mpsks_clients': [MPSKClientEntry(name="Hallo", id=0, mac="11:11:11:11:11")],
```

Comment:
Please use a different test mac (e.g. `00:de:ad:be:ef:00`). This one is invalid in two ways:
- It has only 5 octets instead of 6
- It has the multicast bit set.